test/suites/network/multi-nic.robot

```diff
@@ -79,24 +132,21 @@ Initialize Nmcli Variables
     ${stdout}    ${stderr}    ${rc}=    Execute Command
     ...    ${NMCLI_CMD} | head -1
     ...    return_stdout=True    return_stderr=True    return_rc=True
-    Log    ${stderr}
     Should Be Equal As Integers    ${rc}    0
     Set Suite Variable    \${NIC1_NAME}    ${stdout}
 
     ${stdout}    ${stderr}    ${rc}=    Execute Command
     ...    ${NMCLI_CMD} | tail -1
     ...    return_stdout=True    return_stderr=True    return_rc=True
-    Log    ${stderr}
     Should Be Equal As Integers    ${rc}    0
     Set Suite Variable    \${NIC2_NAME}    ${stdout}
 
-Nmcli Connection Control
+Disable Interface
     [Documentation]    Run nmcli connection command with arguments
```

Comment:
Documentation went out of sync a little bit. Also: should we include information that this change is not persistent, i.e. reversible by reboot?

Reply:
Done